jina/clients/base/grpc.py

```diff
@@ -71,6 +137,7 @@ async def _get_results(
         max_backoff: float = 0.1,
         backoff_multiplier: float = 1.5,
         results_in_order: bool = False,
+        stream: bool = True,
```

Comment:
shouldn't it be False by default?

Reply:
I wanted to get common understanding and consensus before making the unary call default. If there are no objections or doubt I will make the unary call default and update the documentation.

Reply:
put this default to False as the public API calling?